Simplified/Audiobooks/DPLA/JWKResponse.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  JWKResponse.swift
+//  Simplified
+//
+//  Created by Vladimir Fedorov on 31.08.2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+import NYPLAudiobookToolkit
+
+struct JWKResponse: Codable {
+  let keys: [JWK]
+}
+
+struct JWK: Codable {
+  /// Custom JWK structure in response
+  private enum JWKKeys: String, CodingKey {
+    case kty
```

Comment:
is this key type? can you spell it out, otherwise it's cryptic.

Reply:
`kty` is a required JWK field. I removed the rest of the fields since we are not using them anymore, leaving this one as it was used in a function that was later also removed. I've removed `kty` as it is not in use anymore.